packages/react-notifications/src/components/notification/notification.tsx

```diff
@@ -82,8 +81,8 @@ const defaultProps: Pick<Props, 'fixed'> = {
 const Notification = (props: Props) => {
   const intl = useIntl();
   const id = useFieldId(undefined, sequentialId);
-  const { isNewTheme, themedValue } = useTheme();
-  const Button = themedValue(IconButton, SecondaryIconButton);
+  const { isNewTheme } = useTheme();
+  const Button = SecondaryIconButton;
```

Comment:
My first idea was to not change this ones and keep the `themedValue`. Then I refactored the codemod and they started to get updated.

The ideal scenario would be your second option, but it looked a bit complex to me so I thought I could leave it like this and adjust manually in the PR where we run the codemod.
Let me give it another look to see if I can find an easy implementation for option 2.

Reply:
Ok, thanks for looking into it.

If it's too complex let's leave it as-is and adjust manually in case. I don't recall using this pattern too often anyway.

Reply:
Ok, I updated the codemod to handle these use cases and it works 99% of the time. I only found an edge case where it will need an easy to spot manual fix after running it.

Modified in this commit: 06946f5981935a2197df06a6ee75d1d5cd65ca41

Reply:
Super nice, thanks! 🥳